### ISSUE TITLE
HTML builder: include with literal tag does not preserve quotes

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#299]
The "include" directive with the ":literal:" argument does not preserve quotes as straight quotes.

See http://groups.google.com/group/sphinx-dev/browse_thread/thread/35b8071ffe9a8feb

1) Source text snippet:

```
differences between include and literalinclude
==============================================

.. literalinclude:: jabberwocky1.txt

.. include:: jabberwocky1.txt
   :literal:
   :tab-width: 4 
```

2) Included file:

```
'Twas brillig, and the slithy toves
    Did gyre and gimble in the wabe;
        All mimsy were the borogoves,
            And the mome raths outgrabe.

Bingo "Bingo" 'Bingo' test for quotes.
```

3) output HTML fragment:

```
<h1>differences between include and literalinclude<a class="headerlink" href="#differences-between-include-and-literalinclude" title="Permalink to this headline">¶</a></h1>
<div class="highlight-python"><pre>'Twas brillig, and the slithy toves
    Did gyre and gimble in the wabe;
        All mimsy were the borogoves,
            And the mome raths outgrabe.

Bingo "Bingo" 'Bingo' test for quotes.</pre>
</div>

<pre class="literal-block">
&#8216;Twas brillig, and the slithy toves
    Did gyre and gimble in the wabe;
        All mimsy were the borogoves,
            And the mome raths outgrabe.

Bingo &#8220;Bingo&#8221; &#8216;Bingo&#8217; test for quotes.
</pre>
</div>
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/299
- Originally reported by: Jason S
- Originally created at: 2009-12-10T19:09:36.890

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-12-24 12:19:24+00:00_

Thanks, fixed in <<cset 88e54f951bb7c11eb1acebe7e0d242a56a0fb437>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:33+00:00_

Removing version: 0.6.3 (automated comment)
